app/views/interactions/index.html.erb

```diff
@@ -43,7 +43,13 @@
           </p>
         </td>
         <td class='text-center'>
-          <%= link_to(interaction.button_text, edit_link_path(interaction_slug: interaction.slug), class: "btn btn-default btn-s") %>
+          <div class='actions add-vertical-margins col-xs-6 no-gutter'
+               data-module="track-click"
+               data-track-action="editButtonClick"
+               data-track-label="<%= edit_link_path(interaction_slug: interaction.slug) %>"
+               data-track-category="userInteraction:LLM">
+               <%= link_to(interaction.button_text, edit_link_path(interaction_slug: interaction.slug), class: "btn btn-default btn-s js-track") %>
```

Comment:
This is what the div wrapping was for (red Delete button)
<img width="1216" alt="screen shot 2017-04-12 at 17 49 36" src="https://cloud.githubusercontent.com/assets/1354439/24969647/5402f1a6-1fa9-11e7-911f-2d224fe51fc7.png">

This wasn't necessary on the index pages which looks like this 
<img width="1243" alt="screen shot 2017-04-12 at 17 56 48" src="https://cloud.githubusercontent.com/assets/1354439/24969671/6ef0572e-1fa9-11e7-94ab-f133e74a8ffd.png">